src/domain/synthetics/markets/useGlvMarkets.ts

```diff
@@ -275,11 +260,12 @@ export function useGlvMarketsInfo(
             shiftLastExecutedAt: data[glv.glvToken + "-info"].glvShiftLastExecutedAt.returnValues[0],
             shiftMinInterval: data[glv.glvToken + "-info"].glvShiftMinInterval.returnValues[0],
             isGlv: true,
-            indexToken: indexToken,
+            glvToken: glvToken,
+            indexToken: glvToken,
```

Comment:
glvs don't have indexToken. can we elliminate it? same for indexTokenAddress

Reply:
removed